src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -976,77 +914,34 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
     /**
      * Returns true if our membership event needs to be updated
      */
-    private membershipEventNeedsUpdate(
-        myPrevMembershipData?: CallMembershipData,
-        myPrevMembership?: CallMembership,
-    ): boolean {
-        if (myPrevMembership && myPrevMembership.getMsUntilExpiry() === undefined) return false;
-
-        // Need to update if there's a membership for us but we're not joined (valid or otherwise)
-        if (!this.isJoined()) return !!myPrevMembershipData;
-
-        // ...or if we are joined, but there's no valid membership event
-        if (!myPrevMembership) return true;
-
-        const expiryTime = myPrevMembership.getMsUntilExpiry();
-        if (expiryTime !== undefined && expiryTime < this.membershipExpiryTimeout / 2) {
-            // ...or if the expiry time needs bumping
-            this.relativeExpiry! += this.membershipExpiryTimeout;
-            return true;
-        }
-
-        return false;
-    }
-
-    private makeNewMembership(deviceId: string): SessionMembershipData | {} {
-        // If we're joined, add our own
-        if (this.isJoined()) {
-            return this.makeMyMembership(deviceId);
-        }
-        return {};
-    }
-    /**
-     * Makes a new membership list given the old list along with this user's previous membership event
-     * (if any) and this device's previous membership (if any)
-     */
-    private makeNewLegacyMemberships(
-        oldMemberships: CallMembershipData[],
-        localDeviceId: string,
-        myCallMemberEvent?: MatrixEvent,
-        myPrevMembership?: CallMembership,
-    ): ExperimentalGroupCallRoomMemberState {
-        const filterExpired = (m: CallMembershipData): boolean => {
-            let membershipObj;
-            try {
-                membershipObj = new CallMembership(myCallMemberEvent!, m);
-            } catch {
-                return false;
-            }
-
-            return !membershipObj.isExpired();
-        };
+    // private membershipEventNeedsUpdate(
+    //     myPrevMembershipData?: SessionMembershipData,
+    //     myPrevMembership?: CallMembership,
+    // ): boolean {
+    //     if (myPrevMembership && myPrevMembership.getMsUntilExpiry() === undefined) return false;
```

Comment:
Why the commented function?

Reply:
We want to add this to the session membership events. But i will make this PR just about removing pieces. So this if for later.